webpack.config.js

```diff
@@ -8,6 +6,7 @@ const NODE_MODULES = process.env.NODE_MODULES || 'node_modules';
 module.exports = {
     entry: {
         newsroom_js: './assets/index.js',
+        newsroom_images: './assets/images.js',
```

Comment:
Is this needed anymore, considering you're using the `static/` folder on the back-end? If not, then all these changes in `webpack.config.js` can be reverted